### PR TITLE
Update apps and README

### PR DESCRIPTION
**Pull request by drazioti**
1. genume --help changed to python3 -m genume --help
2. added software that rpi uses

### CODE REVIEW

File: scripts/software/apps.sh [hunk 1]
```diff
@@ -237,10 +240,9 @@ check "firefox" "--version"
 
 
 check "thunderbird" "--version"
+check "claws-mail" "--version"
 
 
-# Miscellaneous.
```
Comment (Sima214) [inline scripts/software/apps.sh]: Why did you remove this comment?

Comment (drazioti) [reply scripts/software/apps.sh]: didn't notice
